data/keymaps/i386/azerty/fr-latin9.map

```diff
@@ -418,6 +418,7 @@ keycode 100 = AltGr
 # La touche Ctrl+Pause = Attn = 101 a un code pour elle-m�me
 #
 keycode 101 = Break
+	control keycode 101 = Control_c
```

Comment:
The commit that introduces this change has a misleading commit message.

This change makes `Ctrl`+`Pause/Break` act as Ctrl+C, but the commit message is only about the `Print Screen/SysRq` key.

Do you have information about why `Ctrl`+`Pause/Break` should act as Ctrl+C?

Reply:
Searching more info.

The original bug report from 2001 says:

ncurses client crashes after [PrtScr] press.

On console [PrtScr] is <Ctrl-\> which causes SIGQUIT to be send to the
application. In an xterm [PrtScr] is dead, but SIGQUIT is sent on <Ctrl-\>
too. 

So its original intention was global unmapping of [PrtScr] from <Ctrl-\>, as it caused unintentional interruption of tasks.

[PrtScr] is keycode  99 = Control_backslash

The Ctrl-\ is supposed to send SIGQUIT.
Why it's mapped to PrtScr I don't know.

And regarding the fix:

Hey, sed and bash programming knowledge pays off.
(Other people would probably have found a solution with awk or perl.)

for name in grep -l "^keycode.*99.*Control_backslash"  -r *; 
  do sed -f cvt.sed < $name >$name.new; 
  if diff $name $name.new >/dev/null; then 
    rm -f $name.new; 
  else
    mv -f $name.new $name; 
  fi; 
done

Bug fixed.

The sed script:
```
/^keycode[ 	]*99[ 	=]*Control_backslash/ {
 s/Control_backslash/VoidSymbol/
 n
 /^[ 	]*control[	 ]*keycode[	 ]*99/! {
  i \
	control keycode  99 = Control_backslash
 }
}
/^keycode[ 	]*101[ 	=]*Break/ {
 n
 /^control[ 	]*keycode[ 	]*101/ {
  s/\(101[ 	=]*\)[^ #]*/\1Control_c/
 }
 /^control[ 	]*keycode[ 	]*101/! {
  i \
	control keycode 101 = Control_c
 }
}

```

Reference: https://bugzilla.opensuse.org/show_bug.cgi?id=23380 (SUSE internal link; I can provide more info.)

Reply:
So it looks like a side intention, but the author did not left any comment why he decided to map Ctrl+Break and Shift+Break.